docs/aws-professional/services.md

```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
 
 ## Marketplace
 
-| Area        | AWS service     | Azure service                                                 | Description                                                                                                                                   |
-|-------------|-----------------|---------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
-| Marketplace | AWS Marketplace | [Azure Marketplace](https://azure.microsoft.com/marketplace/) | Easy-to-deploy and automatically configured third-party applications, including single virtual machine or multiple virtual machine solutions. |
+| Area        | AWS service     | Azure service                                                 | Description                                                                                                                                               |
+|-------------|-----------------|---------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Marketplace | AWS Marketplace | [Azure Marketplace](https://azure.microsoft.com/marketplace/) | Easy-to-deploy and automatically configure third- AWS Marketplace Marketplace party applications, including single or multiple virtual machine solutions. |
 
 
 ## Compute
 
-| Area                            | AWS service                     | Azure service                                                                                                                                                                                                                                                                                                                                                                | Description                                                                                                                                                                                                                                     |
+| Service                            | AWS                     | Azure                                                                                                                                                                                                                                                                                                                                                                | Description                                                                                                                                                                                                                                     |
```

Comment:
This column is not really a list of "services" per se - e.g., "backend process orchestration" is not a service. I think that's why the original column heading says "Area"

Reply:
I find "service" appears to be a better description, but we can use either.